tests/util/test_dt.py

```diff
@@ -187,6 +187,7 @@ def test_parse_time_expression():
     assert [x for x in range(24)] == dt_util.parse_time_expression("*", 0, 23)
 
     assert [42] == dt_util.parse_time_expression(42, 0, 59)
+    assert [42] == dt_util.parse_time_expression("42", 0, 59)
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6309587)